package.json

```diff
@@ -9,83 +9,73 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "devDependencies": {
-    "@playwright/test": "^1.53.2",
-    "@testing-library/jest-dom": "^6.6.4",
+    "@playwright/test": "^1.55.1",
+    "@testing-library/jest-dom": "^6.8.0",
     "@testing-library/react": "^16.3.0",
-    "@types/node": "^22.13.14",
-    "@typescript-eslint/eslint-plugin": "^7.15.0",
-    "@typescript-eslint/parser": "^7.15.0",
-    "@vitejs/plugin-react": "^4.5.2",
+    "@types/node": "^24.5.2",
+    "@typescript-eslint/eslint-plugin": "^8.44.1",
+    "@typescript-eslint/parser": "^8.44.1",
+    "@vitejs/plugin-react": "^5.0.4",
     "@vitest/ui": "^3.2.4",
-    "assert": "^2.1.0",
     "autoprefixer": "^10.4.21",
-    "browserify-zlib": "^0.2.0",
-    "del-cli": "^6.0.0",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.5.3",
-    "eslint-plugin-react": "^7.37.4",
-    "events": "^3.3.0",
+    "del-cli": "^7.0.0",
+    "eslint": "^9.36.0",
```

Comment:
Upgrading aslant requires a new config file. You could use the one from https://github.com/vega/vega-embed/blob/main/eslint.config.js and remove rules that are not relevant/add other relevant rules.